pkg/client/client.go

```diff
@@ -191,16 +192,7 @@ func (n *NutanixClientHelper) buildProviderFromFile() (envTypes.Provider, error)
 }
 
 func Build(creds prismgoclient.Credentials, additionalTrustBundle string) (*nutanixClientV3.Client, error) {
-	cli, err := buildClientFromCredentials(creds, additionalTrustBundle)
-	if err != nil {
-		return nil, err
-	}
-	// Check if the client is working
-	_, err = cli.V3.GetCurrentLoggedInUser(context.Background())
-	if err != nil {
-		return nil, fmt.Errorf("failed to get current logged in user with client: %w", err)
-	}
-	return cli, nil
```

Comment:
With session-auth, this happens automatically as basic auth credentials are used to generate a session cookie by making a GetCurrentLoggedInUser call internally.